packages/appium/docs/en/guides/caps.md

```diff
@@ -2,9 +2,150 @@
+| `platformName`                        | `string`  | Required  | The type of platform hosting the app or browser                                                                                                                 |
+| `appium:automationName`               | `string`  | Required  | The name of the Appium driver to use                                                                                                                            |
+| `browserName`                         | `string`  | Optional  | The name of the browser to launch and automate, if the driver supports web browsers as a special case                                                           |
+| `appium:app`                          | `string`  | Optional  | The path to an installable application                                                                                                                          |
+| `appium:deviceName`                   | `string`  | Optional  | The name of a particular device to automate, e.g., `iPhone 14`                                                                                                  |
+| `appium:platformVersion`              | `string`  | Optional  | The version of a platform, e.g., for iOS, `16.0`                                                                                                                |
+| `appium:newCommandTimeout`            | `number`  | Optional  | The number of seconds the Appium server should wait for clients to send commands before deciding that the client has gone away and the session should shut down |
+| `appium:noReset`                      | `boolean` | Optional  | If true, instruct an Appium driver to avoid its usual reset logic during session start and cleanup (default `false`)                                            |
+| `appium:fullReset`                    | `boolean` | Optional  | If true, instruct an Appium driver to augment its usual reset logic with additional steps to ensure maximum environmental reproducibility (default `false`)     |
+| `appium:eventTimings`                 | `boolean` | Optional  | If true, instruct an Appium driver to collect [Event Timings](./event-timing.md) (default `false`)                                                              |
+| `appium:printPageSourceOnFindFailure` | `boolean` | Optional  | If true, collect the page source and print it to the Appium log whenever a request to find an element fails (default `false`)                                   |
+
+
+Some drivers place more complex constraints on capabilities as a group. For example, while the
+`appium:app` and `browserName` capabilities are listed above as optional, the XCUITest driver
+requires that at least one of `appium:app`, `browserName`, or `appium:bundleId` are included in the
```

Comment:
this is not quite true. It is possible to start a no-app session on device home screen if none of these caps is provided